scalameta/transversers/shared/src/main/scala/scala/meta/transversers/SimpleTraverser.scala

```diff
@@ -0,0 +1,6 @@
+package scala.meta
+package transversers
+
+private[meta] class SimpleTraverser {
```

Comment:
I will follow up with a PR to make this class public so that users can extend it to implement custom traversals like documented here https://scalameta.org/docs/trees/guide.html#custom-traversals